docs/docsets/Swifty.xml

```diff
@@ -1 +1 @@
-<entry><version>1.2.0</version><url>https://flipkart.github.io/docsets/Swifty.tgz</url></entry>
+<entry><version>1.3.0</version><url>https://flipkart.github.io/docsets/Swifty.tgz</url></entry>
```

Comment:
This doesn't need to edited manually, these docs are generated by [Jazzy](https://github.com/realm/jazzy).